### PR TITLE
Chore create release notes 0.1.0

### PR DESCRIPTION
**Pull request by Szymiks**
Resolves: https://github.com/Code-Poets/sheetstorm/issues/27

### CODE REVIEW

File: NOTES-RELEASE.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+###v0.1.0
+ - Determine programming tools and requirements.
```
Comment (kbeker) [inline NOTES-RELEASE.md]: This is unnecessary here.

File: NOTES-RELEASE.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+ - Create graphical base user interface.
+ - Implement sheetstorm user authentication.
+ - Entering the possibility of changing the password for users.
+ - Users are allow to join project.
+ - Show accurate hour display
+ - Description field is ready to prepare reports.
+ - Admin is allow to manage employees list.
+ - Manager is allow to see projects and manage them.
+ - Markdown support to description field.
+ - Users are possible to set and change account details.
```
Comment (kbeker) [inline NOTES-RELEASE.md]: Also please add to all this notes numbers of pull requests where it is from.

File: NOTES-RELEASE.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+
+###v0.1.0
```
Comment (kbeker) [inline NOTES-RELEASE.md]: There is missing space between `###` and number version

File: NOTES-RELEASE.md (hunk 2 above)
Comment (Karrp) [reply NOTES-RELEASE.md]: How should it look like?

Something like that?
> - Markdown support to description field. #37

Comment (kbeker) [reply NOTES-RELEASE.md]: > How should it look like?
> 
> Something like that?
> 
> > * Markdown support to description field. #37

Almost yes :) I would like to have it written with brackets, like this:

``` * Markdown support to description field. (#37) ```

File: NOTES-RELEASE.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+
```
[the remark this answers is not in the document]
Comment (Karrp) [reply NOTES-RELEASE.md]: Done

File: NOTES-RELEASE.md (hunk 3 above)
Comment (Karrp) [reply NOTES-RELEASE.md]: Done

File: NOTES-RELEASE.md (hunk 2 above)
Comment (Karrp) [reply NOTES-RELEASE.md]: I will try to do it tomorrow ;)

**Pull request merged**